src/transform/utils.js

```diff
@@ -0,0 +1,57 @@
+/**
+*
+* @licstart  The following is the entire license notice for the JavaScript code in this file.
+*
+* Publication archives record transformer for the Melinda record batch import system
+*
+* Copyright (C) 2019-2020 University Of Helsinki (The National Library Of Finland)
```

Comment:
Check that all Copyrights match

Reply:
Fixed in https://github.com/NatLibFi/melinda-record-import-transformer-publication-archives/commit/870210223b2198e6f4e2a851a87441ad6a551b64